src/renderer/viz/expressions/histogram/Histogram.js

```diff
@@ -90,6 +82,7 @@ export default class Histogram extends BaseExpression {
     }
 
     _getBucketsValue ([...histogram], buckets) {
+        buckets = buckets.length === undefined ? this._genBreakpoints(buckets) : buckets;
```

Comment:
`.length === undefined ?`    Is then buckets an object been initialized before and not a simple array? From outside I was expecting something like `buckets === undefined```...

Reply:
That's because the `buckets` parameter can be a number or an array, i'm going to change the condition to make it clearer